model/ftn/w3odatmd.ftn

```diff
@@ -143,8 +144,10 @@
 !      IRQGO     I.A.  Public   Array with MPI handles W3IOGO.
 !      FLOGRD    L.A.  Public   FLags for output fields.
 !      FLOGR2    L.A.  Public   FLags for coupling fields.
+!      FLOGRR    L.A.  Public   FLags for coupling restart.
 !      FLOGD     L.A.  Public   Flags for output groups
 !      FLOG2     L.A.  Public   Flags for coupling groups
+!      FLOGR     L.A.  Public   FLags for coupling restart.
```

Comment:
This is the 1D array for the restart flags? Maybe clarify in the comment to make distinct from FLOGRR? e.g.
Flags for coupling restart (1D)